mizar/dp/mizar/workflows/builtins/pods/create.py

```diff
@@ -170,6 +165,11 @@ def run(self):
             self.finalize()
             return
 
+        n = net_opr.store.get_net(spec['subnet'])
```

Comment:
Thanks for figuring this out so fast.

I'm wondering if we should have a separate update.py file for on.update trigger, that will make it a lot cleaner to write future code. (not now - post 1/30 if we get to it)